patches/expo-image+2.0.4.patch

```diff
@@ -1,3 +1,21 @@
+diff --git a/node_modules/expo-image/android/build.gradle b/node_modules/expo-image/android/build.gradle
+index a85fb5e..ac568c1 100644
+--- a/node_modules/expo-image/android/build.gradle
++++ b/node_modules/expo-image/android/build.gradle
+@@ -44,7 +44,12 @@ dependencies {
+   kapt "com.github.bumptech.glide:compiler:${GLIDE_VERSION}"
+   api 'com.caverock:androidsvg-aar:1.4'
+ 
+-  implementation "com.github.penfeizhou.android.animation:glide-plugin:3.0.2"
++  implementation project(':frameanimation')
++  implementation project(':gif')
++  implementation("com.github.penfeizhou.android.animation:glide-plugin:3.0.2") {
++    exclude group: 'com.github.penfeizhou.android.animation', module: 'frameanimation'
++    exclude group: 'com.github.penfeizhou.android.animation', module: 'gif'
++  }
```

Comment:
The fixes are in [frameanimation](https://github.com/rahimrahman/APNG4Android/commit/74dd31cf4363b012c7d81fb8356f85d96d29ff24#diff-2b65779ce0c127ece88ea8df6a20a44a7f972b91a43b43094927d20a2d64ee59) & [gif](https://github.com/rahimrahman/APNG4Android/commit/74dd31cf4363b012c7d81fb8356f85d96d29ff24#diff-03495d8305cd255ff5c38eba5b5d8f43777faf05e02f41c39eb9c826f871b0d1).